Formula/butteraugli.rb

```diff
@@ -0,0 +1,20 @@
+class Butteraugli < Formula
+  desc "Measuring differences between images."
+  homepage "https://github.com/google/butteraugli"
+  url "https://github.com/google/butteraugli/archive/2f8d6a0b1466b003d610271ef1f0ae3dd0d28998.tar.gz"
+  version "0.1"
```

Comment:
Yes, it's true that we haven't settled on a solution yet. It's a bit of a delicate matter because whether a formula needs to be upgraded is immediately tied to its version number. That's also the reason why we can't just use the commit ID, as we need something that we can compare numerically and put in an order that makes sense (this probably applies to all package managers).

We normally have a fairly strict requirement that upstream tag releases and that we only use tagged releases (though exception to this rule may have been made, even more so in the past). But we're obviously unlikely to be able to convince Google, if that's a policy across all their projects, and the project is sufficiently popular that we probably want to have it in Homebrew proper. So that calls for some pragmatic solution, that we haven't found yet.